devel/mimalloc/Portfile

```diff
@@ -23,3 +27,37 @@ checksums           rmd160  362b6ba6270889af10af46e41e469847a42b99a0 \
                     size    1163868
 
 patchfiles          patch-cmake.diff
+
+platform darwin powerpc {
+    patchfiles-append \
+                    0001-prim.h-support-tls_slot-for-PPC.patch \
+                    0002-Easy-on-tests-for-PPC.patch
+}
+
+compiler.cxx_standard   2017
+compiler.c_standard     2011
+
+configure.args-append \
+                    -DMI_BUILD_TESTS=ON
+
+if {${os.platform} eq "darwin" && ${os.major} < 11} {
+    # Turning on MI_OSX_INTERPOSE breaks the build with undefined symbols:
+    # https://trac.macports.org/ticket/67220
+    # Turning on MI_OSX_ZONE breaks test-stress with Bus error.
+    configure.args-append \
+                    -DMI_OSX_INTERPOSE=OFF \
+                    -DMI_OSX_ZONE=OFF \
+                    -DMI_USE_CXX=OFF
```

Comment:
If it's not required, remove it.

Reply:
`-DMI_OSX_INTERPOSE=OFF -DMI_OSX_ZONE=OFF` – these two are required, they are not default, and without them build fails (as noted in comments above).